src-docs/src/views/schema/schema_group.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+
+import { OuiSchemaItem } from '../../../../src/components/schema';
+import { OuiFlexGroup, OuiFlexItem } from '../../../../src/components/flex';
+import { OuiSpacer } from '../../../../src/components/spacer';
+// import { OuiPanel } from '../../../../src/components/panel';
+
+export default () => {
+  const actions = [
+    {
+      iconType: 'trash',
+      'aria-label': 'Delete',
+      onClick: () => {},
```

Comment:
It might be nice to have some sort of toast or something as feedback to show example usage. Could be taken care of in a followup

Reply:
Yeah the docs need a follow up for sure. I wasnt even going to include it originally, but left in so that we know what we want to improve. I could add a quick toast here though :)

Reply:
I skipped this for now. Toast is not as straightforward to add here and i dont want to couple another OUI component with this unless necessary.